src/Plugins/ImageProcessor.Web/ImageProcessor.Web.Plugins.AzureBlobCache/AzureBlobCache.cs

```diff
@@ -66,6 +66,21 @@ public class AzureBlobCache : ImageCacheBase
         private string cachedRewritePath;
 
         /// <summary>
+        /// gets reference or creates container
+        /// </summary>
+        /// <param name="cloudBlobClient">cloud blob client</param>
+        /// <param name="containerName">name of container</param>
+        /// <param name="accessType">access permissions ie. public</param>
+        /// <returns>container for cache items</returns>
+        private CloudBlobContainer CreateContainer(CloudBlobClient cloudBlobClient, string containerName, BlobContainerPublicAccessType accessType)
```

Comment:
Again, I agree - should have read your recommendations for contributions more carefully. However, I hope you got the idea ;-)